tests/security/ibmtss/version_check.pm

```diff
@@ -11,16 +11,16 @@ use strict;
 use warnings;
 use testapi;
 use utils 'zypper_call';
+use packages_check;
 
 sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
-    zypper_call('in ibmtss');
-    my $tagert_ver = '1.6.0';
-    my $current_ver = script_output("rpm -q --qf '%{version}\n' ibmtss");
-    record_info("Current ibmtss package version is $current_ver, target version is $tagert_ver");
-    die 'The package version is not updated yet, please check with developer' if ($current_ver lt $tagert_ver);
+    # Version check
+    my $pkg_list = {ibmtss => '1.6.0'};
+    zypper_call("in " . join(' ', keys %$pkg_list));
+    package_upgrade_check($pkg_list);
```

Comment:
According to the previous expected result, when package version is less than the target version, the test will exit with fail. But now it only record a soft failure.  fail should be better?

Reply:
Softfail should be better then exit the test here, most of time we will still have function test after pkg version check. then we may have more test coverage in this case.
As we discussed, currently, we don't have many requirements for failing the test there if pkg version is not updated yet. so I didn't add a new parameter to set it